### DISCUSSION TITLE
start-here/

### DISCUSSION THREAD

**Discussion by giscus** [dragomano/alpinejs-russian#4] [Q&A]
# start-here/

Знакомство с основами Alpine.js

https://alpinejs.dragomano.ru/start-here/

**Comment by Loginovvadimv** [comment]
Хороший перевод

**Reply by dragomano** [reply]
Спасибо

**Comment by bodryakov-web** [comment]
Хотелось бы практических заданий на выполнение, от простого к сложному

**Reply by dragomano** [reply]
Думал об этом, но пока другие дела отвлекают. А вообще лучше сразу выберите для себя какой-нибудь реальный проект для реализации и пилите потихоньку — на практике быстрей освоите.

**Reply by Mary8jk** [reply]
спасибо! чудесная дока

**Comment by tipodanet** [comment]
[Rain](http://rain.zya.me) -- вот я делаю дождь. Потом соединил с прогнозом погоды, вроде работает. 
Слайдерs на Alpine JS и они отлично связываются со скриптом Curtain.JS .

**Reply by dragomano** [reply]
А как выбрать что-то кроме дождливого Лондона?

**Reply by oblominsk** [reply]
все там есть, там иконка для выбора. Проверка на минимализм. Расскажите, как на Alpine сделать так ,  чтобы город у каждого запоминался свой. (это я в логинах запутался)

**Reply by dragomano** [reply]
LocalStorage — https://js.hyperui.dev/examples/storage

**Reply by oblominsk** [reply]
спасибо, вроде работает , у них в примере ))) .  
Этот дождь я тоже нашел где то .  Вообще идея была сделать код, чтобы каждый мог на свой сайт повесить картинку с дождем. А погода --  в нагрузку, дизайну не мешает.

**Comment by V-Kozintsev** [comment]
Классная дока
